docs/docusaurus.config.js

```diff
@@ -125,7 +125,7 @@ const config = {
               items: [
                 {
                   label: 'Blog',
-                  to: 'https://www.permify.co/blog',
+                  to: 'https://www.permify.co/post',
```

Comment:
Not sure about this one, https://www.permify.co/blog is not valid as the blog lives on https://www.permify.co/post The main website also links to https://www.permify.co/post